package.json

```diff
@@ -29,32 +29,46 @@
     "Google Cloud Dataproc API"
   ],
   "scripts": {
-    "test": "c8 mocha",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "lint": "eslint '**/*.js'",
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "lint": "gts fix && eslint --fix samples/*.js",
     "docs": "jsdoc -c .jsdoc.js",
-    "system-test": "mocha system-test/*.js smoke-test/*.js --timeout 600000",
+    "system-test": "mocha build/system-test smoke-test/*.js",
     "fix": "eslint --fix '**/*.js'",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "prelint": "cd samples; npm link ../; npm i"
+    "prelint": "cd samples; npm link ../; npm i",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "prepare": "npm run compile"
   },
   "dependencies": {
-    "google-gax": "^1.7.5",
+    "chai": "^4.2.0",
```

Comment:
Don't think we want this in the top level dependencies

Reply:
I think it might be in the dev dependencies for samples (`samples/package.json`), but not here. The library itself does not use `chai`.

Reply:
Will remove from here.